docs/IE_DG/Extensibility_DG/Extension.md

```diff
@@ -1,29 +1,29 @@
 # Extension Library {#openvino_docs_IE_DG_Extensibility_DG_Extension}
 
 Inference Engine provides an InferenceEngine::IExtension interface, which defines the interface for Inference Engine Extension libraries.
```

Comment:
May be we should wrap all class names with back tick characters "`"?

Reply:
In the website version, they get highlighted anyways, so I think there's no use 
![image](https://user-images.githubusercontent.com/50987643/111300824-8c147500-8662-11eb-830f-f1877e70830d.png)

Reply:
Ok